devices/src/virtio/queue.rs

```diff
@@ -210,7 +210,7 @@ impl Queue {
             error!("virtio queue with invalid size: {}", self.size);
             false
         } else if desc_table
-            .checked_add(desc_table_size)
+            .checked_add(desc_table_size as u64)
```

Comment:
Looks like removing the `as usize` from `let queue_size = self.actual_size() as usize` above would remove some of these `as` usages.